tests/Tests.hs

```diff
@@ -15,11 +15,11 @@
 -- WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 -- License for the specific language governing permissions and limitations
 -- under the License.
---
+
 -- |
--- Module      : Tests
--- Copyright   : (c) 2013-2014 PivotCloud, Inc
--- License     : Apache-2
+-- Module: Main
```

Comment:
The `module` declaration below is for `Main` already, so it might make sense for the metadata here to reflect that.